descriptor.md

```diff
@@ -68,7 +77,7 @@ The _algorithm_ specifies the cryptographic hash function and encoding used for
 
 A digest string MUST match the following [grammar](considerations.md#ebnf):
 
-```
+```text
```

Comment:
I wonder if github markdown handles ebnf instead of just text here (and below)?

Reply:
Thanks, I added this.